src/pyfmi/fmil_import.pxd

```diff
@@ -545,7 +544,7 @@ cdef extern from 'fmilib.h':
     char * fmi1_import_get_GUID(fmi1_import_t *)
 
     #FMI HELPER METHODS (2.0)
-    char * fmi2_get_types_platform()
+    # char * fmi2_get_types_platform()
```

Comment:
Unused; keeping them around though in case we want them in the future

Reply:
Same comment here as above.